cypress/e2e/user-flows/scenario_008.cy.js

```diff
@@ -0,0 +1,110 @@
+import { SimulatorPage } from "../../pages/simulator/simulatorPage";
+import { EditorPage } from "../../pages/simulator/editorPage";
+import { SlicePage } from "../../pages/slices/slicePage";
+import { SLICE_MOCK_FILE } from "../../consts";
+
+const SLICE = {
+  id: "scenario008",
+  name: "Scenario008",
+  library: "slices",
+};
+
+describe("Scenario 008", () => {
+  let slice = new SlicePage();
+  let simulator = new SimulatorPage();
+  let editor = new EditorPage();
+  before(() => {
+    cy.clearProject();
+  });
+
+  beforeEach(() => {
+    cy.setSliceMachineUserContext({});
+  });
+
+  it("creates slice and tests the editor mocks", () => {
+    cy.createSlice(SLICE.library, SLICE.id, SLICE.name);
+
+    slice
+      .addVariation("SecondVariation")
+      .addNewWidgetField("SimpleTextField", "Key Text")
+      .addNewWidgetField("RichTextField", "Rich Text")
+      .addNewWidgetField("NumberField", "Number")
+      .addNewWidgetField("BooleanField", "Boolean")
+      .addNewWidgetField("SelectField", "Select")
+      .addNewWidgetField("ImageField", "Image")
+      .save();
+
+    // force mock value for the boolean and select fields
+    cy.modifyFile(SLICE_MOCK_FILE(SLICE.name), (mock) =>
```

Comment:
oh I understand this issue now, you could check it against the codec ?

Reply:
That is a really good idea to verify our fixtures are always up to date, make a unit test or special test to verify the fixture are always validated.